lib/alerts_viewer_web/controllers/page_html/home.html.heex

```diff
@@ -8,6 +8,9 @@
     <li>
       <.link patch={~p"/bus"}>Bus</.link>
     </li>
+    <li>
```

Comment:
thought: What do you think about making this the first link in the list since it's what we want to direct outside users to?